### DISCUSSION TITLE
Invert icon 'highlight' when state = on

### DISCUSSION THREAD

**Discussion by sydneysider123** [Clooos/Bubble-Card#492] [Questions about config, custom styles and templates]
Hi

The bubble card currently 'highlights' the icon when the entity state == 'On'.

I currently have a door reed that is 'Off' when the door is closed.

Is it possible to apply a style template to 'highlight' the icon when state == 'Off' instead of state == 'On'?

I currently have all my icons change color to orange when my house is armed (i.e. alarm is 'On', doors are 'Off') and because the highlight is not applied to the garage door, the orange color looks dimmer than the other icons that are also set to the same orange color.

An alternative is probably to use a template sensor with inverted outputs, but would really like to avoid having to create another entity that effectively just duplicates (but inverts) the existing entity if possible.

Thanks

**Comment by MrBearPresident** [comment]
Did you solve this?
Can this be closed?
